GRAPHICS/N_0x799017f9e3b10112.md

```diff
@@ -5,17 +5,17 @@ ns: GRAPHICS
 
 ```c
 // 0x799017F9E3B10112
-void _0x799017F9E3B10112(Any p0, Any p1, Any p2, Any p3, Any p4, Any p5, Any p6, Any p7);
+void _0x799017F9E3B10112(float x, float y, float z, float radius, int r, int g, int b, float opacitiy);
```

Comment:
`opacitiy` -> `opacity`?

Reply:
Isn't changing parameter types an ABI breaking change? I'm looking for some clarification before I submit a PR on a separate unnamed native that is using Any instead of Float.

Reply:
Honestly depends on the native, if it requires floats to work, and it's an undocumented native chances are slim that anyone is actually using it in scripts that break like that. Instead if it requires floats instead of ints then usually in languages like C#, you'd be manually invoking the native anyway because there's no other way to do this.

So yes it is a backwards compatibility issue in most cases, but it depends on the native.